src/components/grid/index.test.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import { shallow } from 'enzyme'
+import { FlexColumn } from './index'
+
+describe('The FlexColumn component', () => {
+  const testChild = <p>Testing</p>
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [JSX not allowed in files with extension '.js'](https://app.codacy.com/manual/PrideInLondon/pride-london-web/pullRequest?prid=4658697)